src/UnityPackage/Assets/Web3Unity/Scripts/Scenes/WebLogin.cs

```diff
@@ -22,12 +22,12 @@ public class WebLogin : MonoBehaviour
     void Start()
     {
         // loads the data saved from the editor config
-        projectConfigSO = (ProjectConfigScriptableObject)Resources.Load("ProjectConfigData", typeof(ScriptableObject));
-        PlayerPrefs.SetString("ProjectID", projectConfigSO.ProjectID);
-        PlayerPrefs.SetString("ChainID", projectConfigSO.ChainID);
+        projectConfigSO = ProjectConfigScriptableObject.LoadDefault();
+        PlayerPrefs.SetString("ProjectID", projectConfigSO.ProjectId);
```

Comment:
Do we even need the player prefs any more? I suppose we have to keep them so as to stay backwards-compatible, but we don't need them in the core of the SDK, only the compat layer.

Reply:
we shouldn't need them there if they're being loaded in elsewhere and said new class is being referenced in the appropriate calls.

Reply:
I didn't change anything here as it's out of the scope if this task